.gitignore

```diff
@@ -1 +1 @@
-.env*
+.env
```

Comment:
I added the `*` because I wanted `.env.deployment` in `k8s` to be ignored.

Speaking of `k8s`, I think `k8s/README.md` needs to be updated a bit to be in line with what you're doing in this PR. Maybe you just change references to `env.example` to reference `/app/env.example`.